pom.xml

```
@@ -141,7 +141,7 @@ Copyright (c) 2012 - Jeremy Long
 
         <jetbrains.annotations.version>23.0.0</jetbrains.annotations.version>
 
-        <com.h2database.version>1.4.199</com.h2database.version>
+        <com.h2database.version>2.0.204</com.h2database.version>
```

Use 2.0.206 as that has a fix for CVE-2021-42392 as noted in #3959 
```suggestion
        <com.h2database.version>2.0.206</com.h2database.version>
```